src/core/config.js

```diff
@@ -74,9 +81,23 @@ const resolveConfig = async (config, configPath, argv) => {
     base.node.travis = defaultWithPath(base.node.travis, '.travis.yml');
     base.node.package = defaultWithPath(base.node.package, 'package.json');
   }
+  base.argv = argv;
+  base.forceFlag = argv.force ? '--force' : '--force-with-lease';
```

Comment:
what are the scenario where you would use --force without lease  😂 ❓

Reply:
I had to few time to use it doing testing.